Editor/EditorCore/Experimental.cs

```diff
@@ -17,31 +20,25 @@ internal static bool meshesAreAssets
 
         internal static bool experimentalFeaturesEnabled
         {
-#if PROBUILDER_EXPERIMENTAL_FEATURES
-            get { return true; }
-#else
-            get { return false; }
-#endif
+            get { return s_experimentalFeatureEnabled.value; }
```

Comment:
If the project settings are removed or modified, this value may not reflect reality. That is, whether `PROBUILDER_EXPERIMENTAL_FEATURES` is defined in the compile symbols is the only accurate way to judge.

Rather than trying to keep a `Pref<bool>` in sync with this value, can we add a callback to `Settings Manager` for when a particular settings instance is mass reset and handle it manually?